src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
I believe @pallavit changed it because it was getting flagged by the API validation analyzer.  But that appears to be by design, so a follow-up change should probably revert this line and add an exemption, since it shouldn't end up getting called if ReadFileModern can be used successfully.